tools/clean.mjs

```diff
@@ -14,6 +14,14 @@ import {promisify} from 'util';
 
 const execAsync = promisify(exec);
 
+try {
+  await execAsync('git status');
+} catch (e) {
+  // If `git status` threw an error, we are not in a git repository, bail out.
+  console.log('Not inside a .git repository');
+  process.exit(0);
```

Comment:
Note that I am using `process.exit(0)` here and not `(1)` because I don't consider this to be an error (and there is nothing I could do to fix the error).